projects/adrv9001/common/adrv9001_qsys.tcl

```diff
@@ -116,9 +194,31 @@ add_connection sys_clk.clk_reset axi_dac_dma.s_axi_reset
 add_connection sys_dma_clk.clk axi_dac_dma.m_src_axi_clock
 add_connection sys_dma_clk.clk_reset axi_dac_dma.m_src_axi_reset
 add_connection sys_dma_clk.clk axi_dac_dma.if_m_axis_aclk
-
 add_connection axi_dac_dma.m_axis util_dac_upack.s_axis
 
+# dac-dma2
+
+add_instance axi_dac_dma2 axi_dmac
+set_instance_parameter_value axi_dac_dma2 {ID} {1} 
+set_instance_parameter_value axi_dac_dma2 {DMA_DATA_WIDTH_SRC} {32} 
```

Comment:
This can be 64 bits wide as the HPS AXI interface to avoid insertion of width conversion logic.